app/views/questions/_fields.html.erb

```diff
@@ -28,6 +28,10 @@
     <% end %>
   </div>
 
+  <div>
+    <%= f.field(:audio_prompt, type: :file, read_only: cant_update_core) %>
```

Comment:
Another thing to reflect on -- why are you using `cant_update_core` here? I'm not saying it's wrong or right, I just want you to think about the reasons.

Reply:
I assumed it to be a field we wouldn't want to change without some form of permission. I looked into the ability file and saw that if the question has been published, it can't be edited

Reply:
Nice job checking the ability file! Slight correction though -- if it's published, you can't do something called `update_core` right? Plain `update` is still allowed?

You'll note, for instance, that you can still edit question titles and hints even if the question is published, right?

For this case, since the audio_prompt is kind of like the question title, I think it's ok to allow editing when published.